services/github/README.md

```diff
@@ -89,13 +89,13 @@ docker-compose -f docker-compose-prod.yml run github-service python manage.py te
 
 # Reference
 
-### Kill All Running Docker Instances
+### Kill all the Running Docker Instances
 
 ```bash
 docker stop $(docker ps -a -q)
 ```
 
-### Check Environment on Server
+### Check Environment on the Server
```

Comment:
Check the Environment on the Server

Reply:
Thank you for reviewing! Resolved the issue.